pkgs/by-name/na/nautilus/wrapper.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  makeWrapper,
+  symlinkJoin,
+  nautilus,
+  nautilusExtensions,
+}:
+
+symlinkJoin {
+  name = "nautilus-with-extensions-${nautilus.version}";
+
+  paths = [ nautilus ] ++ nautilusExtensions;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postBuild = ''
+    wrapProgram "$out/bin/nautilus" \
+      --set "NAUTILUS_4_EXTENSION_DIR" "$out/lib/nautilus/extensions-4"
```

Comment:
We might want to also wrap `XDG_DATA_DIRS` so that the wrapper can install `nautilus-python` scripts. Not sure if we should re-use the `nautilusExtensions` argument or create a new one, though.